nautobot/dcim/api/views.py

```diff
@@ -734,7 +735,7 @@ class CableViewSet(StatusViewSetMixin, ModelViewSet):
 #
 
 
-class VirtualChassisViewSet(ModelViewSet):
+class VirtualChassisViewSet(ModelViewSet, NotesViewSetMixin):
```

Comment:
VirtualChassis is a PrimaryModel - this should *probably* become a NautobotModelViewSet.